src/assets/scss/style.scss

```diff
@@ -31,5 +31,6 @@
  * Fields
  */
 @import "fields/repeater";
+@import "fields/nested-repeater";
```

Comment:
I don't think the separate stylesheet it needed. It suggests it's another field type